service/instance_access.py

```diff
@@ -38,14 +38,14 @@ def _remove_instance_access(instance, username, delay=False):
     return async_task
 
 
-def _retry_playbook_with_args(playboook_snapshot):
-    instance = playboook_snapshot.instance
+def retry_playbook_with_args(playbook_snapshot):
```

Comment:
@steve-gregory FYI, changed this to a public function because it is being called from outside the module: https://github.com/steve-gregory/atmosphere/pull/15/files#diff-16438c2a5e87717addf2d5d534390cc7R74